extensions/applicationinsights-dependencies-js/Tests/Unit/src/ajax.tests.ts

```diff
@@ -208,6 +209,59 @@ export class AjaxTests extends AITestClass {
             }
         });
 
+        this.testCaseAsync({
+            name: "Dependencies Configuration: init with cs promise ikey promise",
```

Comment:
This test is probably going to fail if we roll back the mark changes :-( -- @siyuniu-ms we might need for you to add this tests as part of your changes

Reply:
I will comment out this test! and once the change is done, we can uncomment it!

Reply:
I've added this in the task description for reminder.